Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.cpp

```diff
@@ -2542,9 +2542,13 @@ void ResourceLoadStatisticsStore::clear(CompletionHandler<void()>&& completionHa
     updateCookieBlockingForDomains(WTFMove(domainsToBlock), [callbackAggregator] { });
 }
 
-bool ResourceLoadStatisticsStore::areAllThirdPartyCookiesBlockedUnder(const TopFrameDomain& topFrameDomain)
+bool ResourceLoadStatisticsStore::areAllUnpartitionedThirdPartyCookiesBlockedUnder(const TopFrameDomain& topFrameDomain)
 {
+#if HAVE(ALLOW_ONLY_PARTITIONED_COOKIES)
+    if (thirdPartyCookieBlockingMode() == ThirdPartyCookieBlockingMode::All || thirdPartyCookieBlockingMode() == ThirdPartyCookieBlockingMode::AllExceptPartitioned)
```

Comment:
This seems fine, and obviously doesn't break cases that worked with "all" blocking mode. But how do we test partitioned cases?

Reply:
EWS doesn't have support for partitioned cookies right now, so this change won't actually be tested here. We have other internal tests that verify this change.